src/test/scala/com/microsoft/hyperspace/index/LogicalPlanSerDeTests.scala

```diff
@@ -73,6 +78,50 @@ class LogicalPlanSerDeTests extends SparkFunSuite with SparkInvolvedSuite {
     verifyPlanSerde(scanNode, "hadoopFsRelation.plan")
   }
 
+  private def verifyFileFormat(
+      fileFormat: FileFormat,
+      expectSerializationError: Boolean): Unit = {
+    val relation: HadoopFsRelation =
```

Comment:
can we remove the type to be consistent?

Reply:
thanks @imback82 , done